envs/node/transport.js

```diff
@@ -23,12 +37,12 @@ class Transport extends Obfuscated {
     this.socket = net.connect(
       this.dc.port,
       this.dc.ip,
-      this.handleConnect.bind(this)
+      this.handleConnect
```

Comment:
Please stick to the style. Add a commas and semicolons

Reply:
not sure if comma is needed here